docker-compose.test.yml

```diff
@@ -80,6 +80,45 @@ services:
       - SSH_TARGET_PORT=22
       - SSH_TUNNEL_PORT=11111
       - SSH_KEY_FILE=/opt/id_rsa
+      - SSH_KEY= |-
```

Comment:
One always overrides the other.  You cannot test both in the same container.  In fact, you need a third test to test that one does properly override the other.